.github/workflows/release-image.yml

```diff
@@ -14,11 +14,11 @@ jobs:
         uses: actions/checkout@v2
       - name: Set image tag env var
         # Refer https://stackoverflow.com/a/58178121 for git tag extraction.
-        run: echo "OPERATOR_IMAGE_TAG=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+        run: echo "API_MANAGER_IMAGE_TAG=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
```

Comment:
Operator image is a common name for operators: https://github.com/storageos/api-manager/blob/master/Makefile#L3